examples/gradle-kotlin-dsl/diktat-analysis.yml

```diff
@@ -210,6 +210,12 @@
     extendedIndentOfParameters: false
     # If true: if first parameter in parameter list is on the same line as opening parenthesis, then other parameters can be aligned with it
     alignedParameters: true
+    # If true, expression bodies which begin on a separate line are indented
+    # using a continuation indent. The default is false.
+    #
+    # This flag is called CONTINUATION_INDENT_FOR_EXPRESSION_BODIES in IDEA and
+    # ij_kotlin_continuation_indent_for_expression_bodies in .editorconfig.
+    extendedIndentForExpressionBodies: false
```

Comment:
Examples use the latest release of diktat, not snapshot. And after release the updated `diktat-analysis.yml` is copied into examples as a part of workflow. So there is no need to update those configs manually